crowdsec-docs/docs/appsec/installation.md

```diff
@@ -0,0 +1,149 @@
+
+```bash
+mkdir  -p /etc/crowdsec/acquis.d
+cat > /etc/crowdsec/acquis.d/appsec.yaml << EOF
+listen_addr: 127.0.0.1:4242
+appsec_config: crowdsecurity/virtual-patching
+name: myAppSecComponent
+source: appsec
+labels:
+  type: appsec
+EOF
+```
+
+We'll then restart CrowdSec:
+
+```bash
+systemctl restart crowdsec
+```
+
+And you should be able to see CrowdSec starting the AppSec component in the logs (`/var/log/crowdsec.log`):
+
+```
+INFO[2023-12-05 09:16:31] 1 appsec runner to start                      type=appsec
+INFO[2023-12-05 09:16:31] Starting Appsec server on 127.0.0.1:4242/     type=appsec
+INFO[2023-12-05 09:16:31] Appsec Runner ready to process event          type=appsec uuid=3b80fefe-6665-4f81-8567-a2a7f09a706a
+```
+
+As well as actively listening on the specified port:
+
+```bash
+# netstat -laputen | grep 4242    
+tcp        0      0 127.0.0.1:4242          0.0.0.0:*               LISTEN      0          6923691    779516/crowdsec     
+
+```
+
+# Configuration : Remediation component
+
+<!-- @kka fix version -->
+At the time of writting, only the crowdsec nginx bouncer supports native integration with the AppSec component, and requires version >= X.Y.Z.
+
+[If you don't have the bouncer installed, do it now !](https://docs.crowdsec.net/u/bouncers/nginx)
+
+To enable the integration, you simply have to add a `APPSEC_URL` parameter to the existing nginx bouncer remediation configuration :
+
+> /etc/crowdsec/bouncers/crowdsec-nginx-bouncer.conf
+```
+...
+APPSEC_URL=http://127.0.0.1:4242
+...
+```
+
+:warning:*The remediation component uses the same API key for both AppSec and LAPI communication.*  
+*Make sure your bouncer have a valid API and is properly connected to the LAPI via* `sudo cscli metrics`:warning:
+
+We can now restart our remediation component:
+
+```bash
+sudo systemctl restart nginx
+```
+
+# Making sure everything works
+
+For testing purposes, lets trigger a rule that is part of the [CISA Virtual Patching Collection](https://hub.crowdsec.net):
+
+It is trivial, for example, lets trigger the rule for `CVE-2023-42793` by trying to access an URI that ends with `/rpc2`:<!-- @tko : fix link to collec when merged -->
+
+```
+▶ curl -I localhost/rpc2 
+HTTP/1.1 403 Forbidden
+Server: nginx/1.18.0 (Ubuntu)
+Date: Tue, 05 Dec 2023 14:26:03 GMT
+Content-Type: text/html
+Connection: keep-alive
+```
+
+And if we look at it in a browser, the user is presented with the HTML page emitted by the remediation component (that can be customized):
+
+![appsec-denied](/img/appsec_denied.png)
+
+
+# Et Voila !
+
+Your application should now be protected from the most common exploitation attempts. [If you have already enrolled your instance in the console](/docs/next/console/enrollment), you will see alerts appearing there too!
```

Comment:
maybe we could precise that there is nothing more to do ?
will the collection update automatically ?
do I need to care about other collections ?
can you propose a few more stuff to do ? play with the rules ? do some advanced stuff ?